README.md

```diff
@@ -78,6 +78,14 @@ There are many more parameters you can use. Display the usage doc with the follo
 
     GitLink.exe -h
 
+### Native PDBs
+
+Native PDBs (from C++ projects) are supported by using -a option:
+
+    GitLink.exe <nativePdbfile> -a
+
+All source code files from your git depot will be indexed in the PDB.
```

Comment:
"source code" is very hard to define. For instance, you left out all the `*.cs` files. C++ could have other forms of source code you didn't anticipate. 
If you don't want to list every file extension, you could perhaps just say "all known C++ source files."